mpd/README.md

```diff
@@ -3,296 +3,518 @@
 ## Requirements
 
 You only need one or more Music Player Daemons (MPD).
+For more information about MPD and it's commands see http://www.musicpd.org/doc/protocol/command_reference.html
 
 ## Configuration
 
-### plugin.conf (deprecated) / plugin.yaml
-
-```
-[mpd]
-    class_name = MPD
-    class_path = plugins.mpd
-```
+### plugin.yaml
 
 ```yaml
 mpd:
     class_name: MPD
     class_path: plugins.mpd
+    instance: "bathroom"
+    host: "192.168.177.164"
+    port: 6601
+#    cycle: 2
+
+mpd:
+    class_name: MPD
+    class_path: plugins.mpd
+    instance: "kitchen"
+    host: "192.168.177.164"
+    port: 6602
+#    cycle: 2
 ```
 
-### items.conf (deprecated) / items.yaml
+#### instance
+This attribute is mandatory. You have to provide an unique identifier for every instance.
```

Comment:
Is this really useful? In other setup you register the plugin multiple times using the `instance` setting (as mentioned above) and use the settings like `<attribute-name>@<instance-name> = <value>`.

Can this `instance` attribute for an item be replaced with the syntax `<attrib>@<inst>` and using mjulti-instance plugin?

Reply:
Just saw this in the YAML example below: `mpd_status@bathroom: volume` - so seems to be OK then.